### PR TITLE
[WMS][12.0] Add stock_location_attribute (split from stock_location_zone)

### PR DESCRIPTION
**Pull request by grindtildeath**
Followup to #653

### CODE REVIEW

**Comment by rousseldenis** [comment]
@grindtildeath Maybe another name for this module as I don't really like generic names. Maybe more 'functional' oriented one (I don't know which one :smile:  ).

**Review by daramousk — APPROVED** [review]

**Comment by guewen** [comment]
`stock_location_coordinate`? Or `stock_location_position`?

**Comment by jgrandguillaume** [comment]
I personally like stock_location_position or stock_location_map

File: stock_location_attribute/models/stock_location.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockLocation(models.Model):
+
+    _inherit = 'stock.location'
+
+    corridor = fields.Char('Corridor')
```
Comment (jgrandguillaume) [inline stock_location_attribute/models/stock_location.py:13]: help="Define as the street"

File: stock_location_attribute/models/stock_location.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    row = fields.Char('Row')
+    rack = fields.Char('Rack')
```
Comment (jgrandguillaume) [inline stock_location_attribute/models/stock_location.py:15]: help="Define as the house number within the street"

File: stock_location_attribute/models/stock_location.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    level = fields.Char('Level')
```
Comment (jgrandguillaume) [inline stock_location_attribute/models/stock_location.py:16]: help="Define as the floor of the house"

File: stock_location_attribute/models/stock_location.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+from odoo import fields, models
+
+
+class StockLocation(models.Model):
+
+    _inherit = 'stock.location'
+
+    corridor = fields.Char('Corridor')
+    row = fields.Char('Row')
```
Comment (jgrandguillaume) [inline stock_location_attribute/models/stock_location.py:14]: help="Define as the side within the street"

File: stock_location_attribute/models/stock_location.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    rack = fields.Char('Rack')
+    level = fields.Char('Level')
+    posx = fields.Integer('Box (X)')
```
Comment (jgrandguillaume) [inline stock_location_attribute/models/stock_location.py:17]: help="Optional (X) coordinate of the bin if the location is split in several part. e.g. drawer storage"

File: stock_location_attribute/models/stock_location.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    posy = fields.Integer('Box (Y)')
```
Comment (jgrandguillaume) [inline stock_location_attribute/models/stock_location.py:18]: help="Optional (Y) coordinate of the bin if the location is split in several part. e.g. drawer storage"

File: stock_location_attribute/models/stock_location.py [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    posz = fields.Integer('Box (Z)')
```
Comment (jgrandguillaume) [inline stock_location_attribute/models/stock_location.py:19]: help="Optional (Z) coordinate of the bin if the location is split in several part. e.g. storage tray"

**Comment by guewen** [comment]
Replaced by #805. Changes asked here are done for 13.0.

**Pull request closed without merging**